assets/src/block-validation/style.css

```diff
@@ -1,10 +1,10 @@
-.components-button[aria-label="AMP for WordPress"],
-.components-button[aria-label="AMP for WordPress"]:hover,
-.components-button[aria-label="AMP for WordPress"]:focus {
+.components-button[aria-label="AMP"],
+.components-button[aria-label="AMP"]:hover,
+.components-button[aria-label="AMP"]:focus {
 	color: transparent;
 }
 
-.interface-pinned-items .components-button.has-icon[aria-label="AMP for WordPress"] {
+.interface-pinned-items .components-button.has-icon[aria-label="AMP"] {
```

Comment:
I just realized that this was prone to break in the case of "AMP" being translated into another language. It was even worse before when it was "AMP for WordPress", but there is still a chance that "AMP" is localized into another language. Can we limit the selector to just `.amp-toolbar-icon`?

Reply:
I've tried this in 7455c5726500655bc68342d9a4141aa1a3d8e7c4.

Reply:
Looks good!